draft-ietf-openpgp-pqc.md

```diff
@@ -1572,23 +1572,32 @@ longer signature generation time.
+registry group at https://www.iana.org/assignments/openpgp:
 
-- Registry name: `SLH-DSA-SHA2 parameters`
+- Registry name: `OpenPGP SLH-DSA-SHA2 parameters`
 
   Registration procedure: SPECIFICATION REQUIRED [RFC8126]
 
-  Values defined in this document, {{slhdsa-param-sha2}}.
+  The registry contains the values defined in {{slhdsa-param-sha2}} in this document.
 
-- Registry name: `SLH-DSA-SHAKE parameters`
+- Registry name: `OpenPGP SLH-DSA-SHAKE parameters`
 
   Registration procedure: SPECIFICATION REQUIRED [RFC8126]
 
-  Values defined in this document, {{slhdsa-param-shake}}.
+  The registry contains the values defined in {{slhdsa-param-shake}} in this document.
 
-Furthermore IANA will add the algorithm IDs defined in {{kem-alg-specs}} and
-{{sig-alg-specs}} to the  registry `Public Key Algorithms`.
+Furthermore, IANA is requested to add the algorithm IDs defined in {{iana-pubkey-algos}} to the existing registry `OpenPGP Public Key Algorithms`. The field specifications enclosed in brackets for the ML-KEM + ECDH composite algorithms denote fields that are only conditionally contained in the data structure.
+
+
+{: title="IANA updates for registry 'OpenPGP Public Key Algorithms'" #iana-pubkey-algos}
+ID     | Algorithm           | Public Key Format                                                                                                  | Secret Key Format                                                                                                  | Signature Format                                                                                                  | PKESK Format                                                                                                                                                                                    | Reference
+---  : | -----               | ---------:                                                                                                         | --------:                                                                                                          | --------:                                                                                                         | -----:                                                                                                                                                                                          | -----:
+TBD    | ML-KEM-768 + X25519 | 32 octets X25519 public key ({{tab-ecdh-cfrg-artifacts}}), 1184 octets ML-KEM public key ({{tab-mlkem-artifacts}}) | 32 octets X25519 secret key ({{tab-ecdh-cfrg-artifacts}}), 2400 octets ML-KEM secret-key ({{tab-mlkem-artifacts}}) | N/A                                                                                                               | 32 octets ECC ciphertext, 1088 octets ML-KEM ciphertext \[, 1 octet algorithm ID in case of v3 PKESK\], 1 octet length field of value `n`, `n` octets wrapped session key ({{ecc-mlkem-pkesk}}) | {{ecc-mlkem}}
```

Comment:
```suggestion
TBD    | ML-KEM-768 + X25519 | 32 octets X25519 public key ({{tab-ecdh-cfrg-artifacts}}), 1184 octets ML-KEM-768 public key ({{tab-mlkem-artifacts}}) | 32 octets X25519 secret key ({{tab-ecdh-cfrg-artifacts}}), 2400 octets ML-KEM-768 secret-key ({{tab-mlkem-artifacts}}) | N/A                                                                                                               | 32 octets ECC ciphertext, 1088 octets ML-KEM-768 ciphertext \[, 1 octet algorithm ID in case of v3 PKESK\], 1 octet length field of value `n`, `n` octets wrapped session key ({{ecc-mlkem-pkesk}}) | {{ecc-mlkem}}
```

Reply:
Complete reference to ML-KEM parameter set like you did it for ML-DSA.